mycore-base/src/main/java/org/mycore/datamodel/metadata/MCRObjectID.java

```diff
@@ -208,25 +217,25 @@ public static boolean isValid(String id) {
         if (id == null) {
             return false;
         }
-        String mcrId = id.trim();
-        if (mcrId.length() > MAX_LENGTH) {
+        if (id.length() > MAX_LENGTH) {
             return false;
         }
-        String[] idParts = getIDParts(mcrId);
-        if (idParts.length != 3) {
-            return false;
-        }
-        String objectType = idParts[1].toLowerCase(Locale.ROOT).intern();
-        if (!MCRConfiguration2.getBoolean("MCR.Metadata.Type." + objectType).orElse(false)) {
-            LOGGER.warn("Property MCR.Metadata.Type.{} is not set. Thus {} cannot be a valid id", objectType, id);
-            return false;
-        }
-        try {
-            int numberPart = Integer.parseInt(idParts[2]);
-            if (numberPart < 0) {
+        Matcher m = ID_PATTERN.matcher(id);
+        if (m.matches()) {
+            String objectType = m.group(2).toLowerCase(Locale.ROOT).intern();
+            if (!MCRConfiguration2.getBoolean("MCR.Metadata.Type." + objectType).orElse(false)) {
+                LOGGER.warn("Property MCR.Metadata.Type.{} is not set. Thus {} cannot be a valid id", objectType, id);
```

Comment:
Myabe `"Property MCR.Metadata.Type.{} is not set to true. Thus {} cannot be a valid id"`

Reply:
O.K.